checkov/common/util/docs_generator.py

```diff
@@ -60,10 +61,15 @@ def print_checks(frameworks: Optional[List[str]] = None, use_bc_ids: bool = Fals
     print("\n\n---\n\n")
 
 
+def get_check_link(absolute_path: str) -> str:
+    link_start = 'https://github.com/bridgecrewio/checkov/tree/master/checkov'
```

Comment:
can you move it to global scope, under `ID_PARTS_PATTERN` and then change it to something like `CODE_LINK_BASE`